docs/readmes/proposals/p020_n7_Interface_n_network_initiated_session_modification.md

```diff
@@ -0,0 +1,165 @@
+---
+id: p020_n7_Interface_n_network_initiated_session_modification
+title: SMF to PCF Interface and Network Initiated Session Modification
+hide_title: true
+---
+# Overview
+
+*Author: @pmishra-wavelabs*\
+*Last Updated: 07/12/2021*
+
+## Background and Objective 
+This document provides an approach for the following 
+- Implementing N7 interface towards PCF through Federation Gateway
+  -- Implement 5G Services Based Interface (SBI) for Federation Gateway
+  -- Feg Relay for SM Policy control create, update, delete and update-notify
+  -- Feg Session Proxy for SM Policy control create, update, delete and 
+     update-notify 
+- Implementing Network initiated session modification
+  -- Network Initiated Session Policy association update
+  -- Network initiated session Policy association termination
+## Use Cases
```

Comment:
is network_initiated_session_modification_sequence_diagram the only call flow this feature will support?
If not, then we need `create, update, delete and update-notify` call flows too.  Service definition should be planned based on all callflows to implement